exercises/resistor-color-duo/.meta/src/reference/java/ResistorColorDuo.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Example Class
+ */
+public class ResistorColorDuo {
+    public enum Color {
+        black, brown, red, orange, yellow, green, blue, violet, grey, white;
+    };
+
+    int colorCode(String[] colors) {
+        int resistance = 0;
+        resistance += 10 * Color.valueOf(colors[0]).ordinal();
+        resistance += Color.valueOf(colors[1]).ordinal();
+        return resistance;
+    }
```

Comment:
I think this can be simplified. Can you try to shorten this method and eliminate the local variable?

Reply:
Also to be more in line with the exercise description, maybe rename this method to value()?